### ISSUE TITLE
Node.js Foundation Technical Steering Committee (TSC) Meeting 2017-12-27

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#455]
## Time

**UTC Wed 27-Dec-2017 22:00 (10:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 27-Dec-2017 14:00 (02:00 PM) |
| US / Mountain | Wed 27-Dec-2017 15:00 (03:00 PM) |
| US / Central  | Wed 27-Dec-2017 16:00 (04:00 PM) |
| US / Eastern  | Wed 27-Dec-2017 17:00 (05:00 PM) |
| London        | Wed 27-Dec-2017 22:00 (10:00 PM) |
| Amsterdam     | Wed 27-Dec-2017 23:00 (11:00 PM) |
| Moscow        | Thu 28-Dec-2017 01:00 (01:00 AM) |
| Chennai       | Thu 28-Dec-2017 03:30 (03:30 AM) |
| Hangzhou      | Thu 28-Dec-2017 06:00 (06:00 AM) |
| Tokyo         | Thu 28-Dec-2017 07:00 (07:00 AM) |
| Sydney        | Thu 28-Dec-2017 09:00 (09:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2017-12-27&iso=20171227T22
* or http://www.wolframalpha.com/input/?i=10PM+UTC%2C+Dec+27%2C+2017+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1aDXoq3FH1sJ6_zqeo8s1EGD_QdKNVZV54jxCf0E2x90/edit>

## Agenda

Extracted from **TSC-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/TSC

* Mentorship initiative ramping up. [#443](https://github.com/nodejs/TSC/issues/443)
* Enterprise Advisory Group [#431](https://github.com/nodejs/TSC/issues/431)
* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
* Document guidelines for accountability expectations for all members of the TSC and CommComm [#311](https://github.com/nodejs/TSC/issues/311)

### nodejs/admin

* GitHub Owner permissions [#33](https://github.com/nodejs/admin/issues/33)

## Invited

* Anna Henningsen @addaleax (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Daniel Bevenius @danbev (TSC)
* Evan Lucas @evanlucas (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC)
* Josh Gavant @joshgav (TSC)
* Fedor Indutny @indutny (TSC)
* James M Snell @jasnell (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Brian White @mscdex (TSC)
* Myles Borins @MylesBorins (TSC)
* Ali Ijaz Sheikh @ofrobots (TSC)
* Rod Vagg @rvagg (TSC)
* Michaël Zasso @targos (TSC)
* Sakthipriyan Vairamani @thefourtheye (TSC)
* Trevor Norris @trevnorris (TSC)
* Rich Trott @Trott (TSC)

## Notes

The agenda comes from issues labelled with `TSC-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Uberconference; participants should have the link & numbers, contact me if you don't.

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for that

**Comment by mcollina** [comment]
I thought we were skipping... if not, can we skip?

In any case, I won’t be joining.

**Comment by Trott** [comment]
Adding @danbev.

**Comment by MylesBorins** [comment]
I likely won't be able to make the next two meetings as I'm on Vacation

**Comment by fhinkel** [comment]
Sorry, I can't make it this week

**Comment by rvagg** [comment]
I'm around but would love to skip to ease the end of year load. Anyone opposed to skipping? The agenda is pretty light on urgency.

**Comment by ChALkeR** [comment]
I won't be able to make it, too late in local time (as speciefied in the spreadsheet :wink: ).

**Comment by cjihrig** [comment]
I can make it, but would prefer to skip.

**Comment by jasnell** [comment]
+1 to skipping

**Comment by danbev** [comment]
I'll be around but no problem skipping either.

**Comment by evanlucas** [comment]
Skipping sounds good to me

**Comment by Trott** [comment]
TL;DR: I think we should skip this week too.

GIven the absences, there's almost nothing on the agenda that can be handled anyway.

* Mentorship initiative ramping up. #443

Looks like an FYI only. @mhdawson put it on the agenda and he will miss this meeting.

* Enterprise Advisory Group #431

Issue requires @jasnell. Last I heard, he won't make a TSC meeting until January.

* Document guidelines for accountability expectations for all members of the TSC and CommComm #311

@MylesBorins put this on the agenda and he is likely missing this meeting. I have an action item there and I'll update the GitHub issue on that. No need to meet just so I can give the tiny update to only the people that can/will attend the meeting today.

* GitHub Owner permissions https://github.com/nodejs/admin/issues/33

This is one I added. It might be the only one we can reasonably discuss, but there's no reason (other than that discussion stalled) that discussion can't happen in the GitHub issue instead. I'll open a pull request to "fix" that issue.

**Comment by Trott** [comment]
TL;DR: Skipping! See you next week!

11 of 21 TSC members have indicated that they will miss the meeting (including @mhdawson), that skipping is desirable, or that skipping is acceptable acceptable. We have no one who has said that they would prefer we meet today or that there is a critical issue that needs to be discussed today. I'm going to close this issue. Feel free to comment and re-open if you disagree with this!
